### PR TITLE
static image progress

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: src/components/indexHero.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const HeroInnerContainer = styled.div`
 `;
 
 const Heading = styled.h1`
-    ${tw`text-ts-h1 font-wt-bold mb-6`}
+    ${tw`text-ts-h1 font-wt-bold mb-6 mt-24`}
```
Comment (asasmith) [inline src/components/indexHero.js:18]: We should change this text-color to be `text-brand-1`

File: src/components/fullWidthImageWithText.js [hunk 2]
```diff
@@ -33,6 +34,7 @@ export default function FullWidthImageWithText({
     return (
         <StyledSection>
             {hasParallelTransition && <ParallelTransition />}
+            {hasTopTransition && <ParallelTransition />}
```
Comment (asasmith) [inline src/components/fullWidthImageWithText.js:37]: Let's pick one or the other, we don't need both.

File: src/components/indexHero.js (hunk 1 above)
Comment (kcbreuil) [reply src/components/indexHero.js:18]: @asasmith Should I also change the althero as well?

File: src/pages/index.js [hunk 3]
```diff
@@ -38,14 +38,14 @@ const content = {
...
-    const { heroImage } = data;
+    const { heroImage, accImage } = data;
 
     return (
         <Layout isIndexPage>
             <EventList />
             <IndexHero heroImage={heroImage} cta={content.heroLink} />
             <IndexSubHero heading={content.subHero.heading} body={content.subHero.body} /> 
-            <FullWidthImageWithText heading={content.fullWidthImage.heading} body={content.fullWidthImage.body} image={heroImage} hasParalledTransition />
+            <FullWidthImageWithText heading={content.fullWidthImage.heading} body={content.fullWidthImage.body} image={accImage} hasParalledTransition hasTopTransition />
```
Comment (asasmith) [inline src/pages/index.js:48]: if you use the `hasParallelTransition` you have a typo here (`hasParalledTransition`). Same comment as above, let's pick one and remove the other.

File: src/components/fullWidthImageWithText.js [hunk 4]
```diff
@@ -52,4 +55,5 @@ FullWidthImageWithText.propTypes = {
     image: PropTypes.object.isRequired,
     hasCurvedTransition: PropTypes.bool,
     hasParallelTransition: PropTypes.bool,
+    hasTopTransition: PropTypes.bool,
```
Comment (asasmith) [inline src/components/fullWidthImageWithText.js:58]: whichever one you remove, also remove the proptype

File: src/components/indexHero.js (hunk 1 above)
Comment (asasmith) [reply src/components/indexHero.js]: Actually, sorry, can you move `text-brand-1` to `HeroInnerContainer`. That way the body copy picks up that color too.

Comment (kcbreuil) [reply src/components/indexHero.js]: @asasmith done!

**Pull request merged**